src/dmd/mtype.d

```diff
@@ -5143,11 +5143,13 @@ extern (C++) final class TypeTraits : Type
  */
 extern (C++) final class TypeMixin : Type
 {
+    Loc loc;
```

Comment:
Please update the headers to reflect this change.

Reply:
The class is not in the header.